cms/djangoapps/contentstore/tests/test_orphan.py

```diff
@@ -45,26 +58,107 @@ def setUp(self):
         self.store.update_item(orphan_vertical, self.user.id)
 
         # create an orphaned html module
-        orphan_html = self.store.create_item(self.user.id, self.course.id, 'html', "OrphanHtml")
+        orphan_html = self.store.create_item(self.user.id, course.id, 'html', "OrphanHtml")
         self.store.publish(orphan_html.location, self.user.id)
 
-        self.store.create_child(self.user.id, self.course.location, 'static_tab', "staticuno")
-        self.store.create_child(self.user.id, self.course.location, 'about', "overview")
-        self.store.create_child(self.user.id, self.course.location, 'course_info', "updates")
+        self.store.create_child(self.user.id, course.location, 'static_tab', "staticuno")
+        self.store.create_child(self.user.id, course.location, 'course_info', "updates")
+
+        return course
+
+    def create_course_with_orphan_on_branch(self, branch, DOC):
```

Comment:
Nit: `direct_only` might be clearer. `DOC` seemed like a typo to me at first, since it's all caps.